nvflare/private/fed/client/client_runner.py

```diff
@@ -473,19 +478,108 @@ def fetch_and_run_one_task(self, fl_ctx) -> (float, bool):
         if cookie_jar:
             task_reply.set_cookie_jar(cookie_jar)
 
-        reply_sent = self.engine.send_task_result(task_reply, fl_ctx)
-        if reply_sent:
-            self.log_info(fl_ctx, "result sent to server for task: name={}, id={}".format(task.name, task.task_id))
-        else:
-            self.log_error(
-                fl_ctx,
-                "failed to send result to server for task: name={}, id={}".format(task.name, task.task_id),
-            )
+        self._send_task_result(task_reply, task.task_id, fl_ctx)
```

Comment:
Do we need to add a check here for the return value to see if result was sent or not? As it could return false from ask_to_stop or TASK_GONE

Reply:
We could but not necessary since the method already logged detailed message. In fact, it doesn't need to return anything.